File: Quadrant Selection/QuadrantSelection.py
# Tyler Huffman
# 2020-03-04
# https://open.kattis.com/problems/quadrant

from sys import stdin
# Grab the (x,y) values
x,y = (int(z) for z in stdin.readlines())

# Use basic algebra to determine which quadrant an (x,y) coords are in
if x > 0:
    if y > 0:
        print(1)
    else:
        print(4)
else:
    if y > 0:
        print(2)
    else:
        print(3)

File: Quadrant Selection/QuadrantSelection.cpp
/*
 Tyler Huffman
 2020-03-08
 https://open.kattis.com/problems/quadrant
*/
#include <iostream>
using namespace std;

int main(){
	int x,y;   // Initialize integers to hold coords
	cin >> x;  // Grab the x-coord
	cin >> y;  // Grab the y-coord
	cout << ((x > 0) ? (y > 0) ? "1": "4" : (y < 0) ? "3" : "2"); //Condense the if/else statement into a ternary
		
	return 0;
}

